surface.go

```diff
@@ -185,6 +195,25 @@ func SkipBuggyChecks() BindOption {
 	}
 }
 
+// ImperativeLenses takes a slice of [ImperativeLens]. These lenses will be
+// executed on calls to [Instance.Translate].
+//
+// A mix of Go and CUE lenses may be provided, but only one lens may be provided
```

Comment:
Note: we may need to revisit this text before getting this pull request merged - it doesn't seem to be aligned with what's stated in the pull request description.